tools/provision/darwin.sh

```diff
@@ -18,6 +18,9 @@ function main_darwin() {
 
   brew update
 
+  remove_package libressl
```

Comment:
This seems inefficient to remove libressl and package it again whenever `make deps` run, but it ensures that it builds the bottled version which do not have avx optimizations.